### DISCUSSION TITLE
Is it possible trigger a specific instance of a workflow using Recieve Http Request Activity ?

### DISCUSSION THREAD

**Discussion by nimishdesai1989** [elsa-workflows/elsa-core#349] [Q&A]
I have a long running workflow definition that has a few activities. The workflow hauls at an activity which I want to be triggered with http receive request. But I want to start only specific instance of the workflow and not all others that are in halted state. I have already setup correlation before this activity step. Is it possible to do this without any customisation to the code using the receive http request activity instead of signaled activity ? Any help will be highly appreciated.

**Comment by sfmskywalker** [comment]
Not at this point, but there's an [open issue](https://github.com/elsa-workflows/elsa-core/issues/262) to fix this issue.
For more background, please see [this issue](https://github.com/elsa-workflows/elsa-core/issues/261):

**Reply by sfmskywalker** [reply]
This is now possible. The **HTTP Endpoint** activity (renamed from **Receive Http Request**) can now be triggered using a correlation ID as described in #262
